element_array_ephys/export/nwb/requirements.txt

```diff
@@ -0,0 +1,4 @@
+dandi
+git+https://github.com/catalystneuro/nwb-conversion-tools.git
+git+https://github.com/SpikeInterface/spikeinterface.git
+git+https://github.com/bendichter/python-neo.git@2d0ba5213f6e2add29dd13b2131650e3325edb23
```

Comment:
I don't think these would be installed by installing a local version of the element, would they? Once merged, for example, I think we'd like for someone to be able to
```
git clone https://github.com/datajoint/element-array-ephys.git
pip install ./element-array-ephys
```
and then have access to the export function within their python environment

Reply:
yes, that's the eventual goal. We are working on a release from python-neo which is holding up the release chain that would allow this to be pip installable.

Reply:
Do you have a sense of this timeline? Or is it tough to predict at this point?